### ISSUE TITLE
torchvision transforms to take list of PIL images instead of a single PIL image

### ISSUE THREAD

**Issue by quantummole** [pytorch/vision#610]
For some problems like segmentation we would like the same random transform to be applied on the input and the masks. 
Is it possible to add a check inside the __call__ method to check if the input is a list then apply the same transformation(with same values) for each element of the list

**Comment by sotte** [comment]
[Functional transforms](https://pytorch.org/docs/master/torchvision/transforms.html#functional-transforms) can be reused. It's easy to create transform pipelines for segmentation tasks:
```python
import torchvision.transforms.functional as TF
import random

def my_segmentation_transforms(image, segmentation):
    if random.random() > 5:
        angle = random.randint(-30, 30)
        image = TF.rotate(image, angle)
        segmentation = TF.rotate(segmentation, angle)
    # more transforms ...
    return image, segmentation
```
Maybe not quite as convenient as `Compose` but very flexible. You could write a little function that maps through multiple inputs:
```python
def emap(fn, iterable):
    """eager map because I'm lazy and don't want to type."""
    return list(map(fn, iterable))

sample = image, mask

rotate = partial(TF.rotate, angle=random.randint(-45, 45))
sample = emap(rotate, sample)

resize = partia(TF.resize, size=299)
sample = emap(rotate, sample)
```

**Comment by datumbox** [comment]
This issue appears to be stale, so I'm closing it. If you think it should remain open, feel free to reopen it.
